Makefile.am

```diff
@@ -132,6 +132,7 @@ include include/Makefile.mk
 include javascript/Makefile.mk
 include man/Makefile.mk
 include ola/Makefile.mk
+include olad/plugin_api/Makefile.mk
```

Comment:
Add a comment here as to why this is included at the top level. Normally we just include the Makefiles in the first level of directories.